### PR TITLE
updated proxy version to 1.4.4

### PR DESCRIPTION
**Pull request by rahug20**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Updated gm-proxy version from 1.4.2 to 1.4.4

2. What **changes to custom.yaml** are required?

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Run `make install` and cluster should deploy with the new version successfully.

### CODE REVIEW

**Comment by rahug20** [comment]
close #692

**Comment by tilleryd** [comment]
This aligns the helm-charts with the Grey Matter 1.2 release. 👍

**Comment by tilleryd** [comment]
@rahug20 
```
...
Error: parse error at (secrets/templates/redis-password.yaml:6): function "lookup" not defined

 dtillery@MacBook-Pro-2  ~/dev/workspace/decipher/gm/helm-charts   692-update-to-release-1.2.1  helm version
version.BuildInfo{Version:"v3.0.0", GitCommit:"e29ce2a54e96cd02ccfce88bee4f58bb6e2a28b6", GitTreeState:"clean", GoVersion:"go1.13.4"}
```

**Comment by chrisbsmith** [comment]
@tilleryd , this is a helm version issue. I recommend upgrading to 3.2.4.  We call this out in some places, but we should specify a min version in the [Getting started guide](https://github.com/DecipherNow/helm-charts/blob/release-2.2/docs/Getting%20Started.md#helm)

**Comment by tilleryd** [comment]
@chrisbsmith Yes, there are some assumptions being made. I updated and no longer see the error.

File: fabric/values.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ control:
     # Number of replicas for the deployment
     replicas: 1
     # Version of control
-    version: 1.4.2
+    version: 1.4.4
```
Comment (tilleryd) [inline fabric/values.yaml:79]: This needs to stay at 1.4.2 as there's no 1.4.4 version of control. 

```
  Warning  Failed     10m (x4 over 12m)     kubelet, k3d-greymatter-worker-3  Failed to pull image "docker.production.deciphernow.com/deciphernow/gm-control:1.4.4": rpc error: code = NotFound desc = failed to pull and unpack image "docker.production.deciphernow.com/deciphernow/gm-control:1.4.4": failed to resolve reference "docker.production.deciphernow.com/deciphernow/gm-control:1.4.4": docker.production.deciphernow.com/deciphernow/gm-control:1.4.4: not found
```

**Comment by chrisbsmith** [comment]
@tilleryd i've pushed updates. Can you please review again?

**Comment by tilleryd** [comment]
I had to run catalog-init manually but it doesn't seem to be an issue with this PR.

**Pull request merged**